src/components/shared/Typography/Body2.jsx

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+import tw from 'twin.macro';
+
+export default styled.p`
+  font-style: ${(props) => (props.bold ? 'bold' : 'normal')};
```

Comment:
Change semibold from bold, make sure the things are same as mentioned in the figma file. Do the same for all other fonts.